mango-core/src/main/scala/org/bdgenomics/mango/converters/SAMRecordConverter.scala

```diff
@@ -0,0 +1,223 @@
+/**
+ * Licensed to Big Data Genomics (BDG) under one
```

Comment:
@heuermh this entire file is copied from ADAM

Reply:
Note this isn't fully tested yet, but should be somewhat faster than what is in ADAM
https://github.com/bigdatagenomics/convert/pull/71

Reply:
Awesome thanks @heuermh! Would the flag be added in the convert library?

Reply:
Yes, I could do so.  I'm also considering adding a `convert` adapter layer to the actual converters in ADAM.  That way the implementation classes can continue to be private to ADAM and the convert adapter layer part of the public API.

Reply:
Sorry, been busy with other things. I can implement the `attributes` column flag tomorrow. While thinking about the adapter layer in ADAM I found https://github.com/bigdatagenomics/adam/pull/2156, which needs review and real-life testing to make sure there is no performance regression.